src/Microsoft.Performance.Toolkit.Engine/EngineCreateInfo.cs

```diff
@@ -80,5 +102,26 @@ public string ExtensionDirectory
         ///     behavior.
         /// </summary>
         public VersionChecker Versioning { get; internal set; }
+
+        /// <summary>
+        ///     Gets or sets the name of the runtime on which the application is built.
+        /// </summary>
+        /// <remarks>
+        ///     Defaults to "Microsoft.Performance.Toolkit.Engine".
+        /// </remarks>
+        public string RuntimeName { get; set; } = "Microsoft.Performance.Toolkit.Engine";
```

Comment:
are we able to grab the assembly name dynamically/at compile time instead of hard-coding this?

Reply:
That's probably a good idea.